src/main/java/uk/gov/crowncommercial/dsd/api/auth/logic/AccountResponseProcessor.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.crowncommercial.dsd.api.auth.logic;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.crowncommercial.dsd.api.auth.converter.AccountConverter;
+import uk.gov.crowncommercial.dsd.api.auth.model.spree.account.Account;
+
+@Component
+public class AccountResponseProcessor implements Processor {
```

Comment:
I think you could drop this explicit processor if you apply the `@Converter` annotation to [AccountResponse toAccountResponse](https://github.com/Crown-Commercial-Service/ccs-scale-api-tx-auth-service/pull/1/files#diff-3c1cbea97b70f6c8ba8159ad8eebd5e4ab9d891898d6f04edfb49b0cd520ceaaR18), then you should just be able to call `.convertBodyTo(AccountResponse.class)` - entirely optional though 👍

Reply:
Thanks - will look at including in a follow up PR as part of the tests